server/server.js

```diff
@@ -0,0 +1,32 @@
+const server = require('express')();
+const db = require('../data/dbConfig')
+
+server.use(require('express').json());
+
+server.get('/', (req, res) => {
+    // Sanity Check
+    res.send(`Server Home directory GET is active.`);
+  });
+  
+  server.post('/users', (req, res) => {
+    const newUser = req.body;
+    req.body.name 
+      ? db.insert(req.body)
+          .then(user => {
+            res.status(200).json(user);
+          })
+          .catch(err => {
+            res.status(400).json({ message: 'Failed to add User' });
+          })
+      : res.status(500).json({
```

Comment:
remember not to mix these up! the messages are right, but 400 is bad request on the user's behalf, 500 is server error